google/auth/app_engine.py

```diff
@@ -33,6 +33,21 @@
     app_identity = None
 
 
+def get_project_id():
+    """Gets the project ID for the current App Engine application.
+
+    Returns:
+        str: The project ID
```

Comment:
Can it ever return None? Like if in "flex" and the metadata server is being ornery?